org.eclipse.transformer/src/main/java/org/eclipse/transformer/Transformer.java

```diff
@@ -971,6 +971,11 @@ public boolean setInput() {
 			return false;
 		}
 
+		if (useInputName.isEmpty()) {
```

Comment:
Why not add this to the null check above?

Reply:
I have incorporated empty check into null check.